WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -92,6 +96,7 @@ public void onCreate(Bundle savedInstanceState) {
         restoreSavedInstanceState(savedInstanceState);
         setupActionBar();
         setupRecycleView();
+        initSwipeToRefreshHelper(findViewById(android.R.id.content));
```

Comment:
I noticed that the refresh indicator is not preserved when rotating the device. On slower connections that could be rather confusing so, let's add support for rotation, wdyt? Thanks!